frontend/src/components/pages/CampsList/CampsTable.tsx

```diff
@@ -237,6 +228,16 @@ const CampsTable = ({
                         <Text textStyle="buttonRegular">Edit camp</Text>
                       </PopoverBody>
                     )}
+                    <PopoverFooter
```

Comment:
I put it in the footer so that we get the nice line in-between, but it looks worse when camp is published and there's only one button -- pretty trivial, your call